src/code/providers/google-drive-provider.coffee

```diff
@@ -12,11 +12,27 @@ GoogleDriveAuthorizationDialog = React.createFactory React.createClass
   displayName: 'GoogleDriveAuthorizationDialog'
 
   getInitialState: ->
+    @_hasLoadedGAPI = false
```

Comment:
It's actually the clients that we're interested in, so it's `window._LoadedGAPIClients` that we care about, but using that simplifies the code. I've added another commit which makes the change.